docs/modules/ROOT/pages/references/glossary.adoc

```diff
@@ -1,7 +1,72 @@
 = Glossary
 
-https://azure.microsoft.com/de-de/services/kubernetes-service/[AKS]:: Azure Kubernetes Service (AKS)
+////
+You can link these terms using xrefs and their IDs, for example:
+  xref:reference/glossary.adoc#service-instance[Service Instance]
+////
+
+== General
+
+[[service]]Service::
+A service that can be ordered.
+It is automatically provisioned, can be consumed by a service consumer and generally adheres to our quality requirements.
+
+[[service-consumer]]Service Consumer::
+An application that connects to a service instance as a client.
+
+[[service-order]]Service Order::
+The act of deploying a Kubernetes resource that triggers the provisioning of a new service instance.
+
+[[service-instance]]Service Instance::
+A single isolated occurrence of a service.
+
+[[cloud-instance]]Cloud Instance:::
+An instance of a cloud service offering by a cloud service provider.
+
+[[converged-service-instance]]Converged Service Instance:::
+An instance of a converged service blueprint.
+Either in the same namespace as the service order was made or in another namespace.
+
+== Service Location
+
+Services can be provisioned on various places:
+
+[[converged-cluster]]Converged Cluster::
+Service instance runs on the same cluster as the service consumer.
+
+[[service-cluster]]Service Cluster::
+Service runs on one or more dedicated clusters, reserved for services.
+
+[[cloud-service]]Cloud Service::
+A type of service architecture offered by a cloud service provider such as MS, AWS or GCP.
+
+[[converged-service]]Converged Service::
+A type of service developed to run on a converged cluster.
+
+[[converged-service-architecture]]Converged Service Architectures::
+Architecture refers to the deployment strategy of a service instance in relation to high availability.
+
+[[standalone]]Standalone:::
+A single node instance without any high availability.
+
+[[replicated]]Replicated:::
+The service offers data replication to one or more other instances.
+
+[[clustered]]Clustered:::
+A fully clustered version of the service, usually offering active / active replication with fully automated cluster handling.
```

Comment:
Is there a reason why you copied the full glossary from AppCat? IMHO the appcat glossary doesn't fully apply to APPUiO Managed Kubernetes Services. So I'd opt to remove that again from this document here.

Reply:
Yep, that was confusing. Up until now, I had the impression to read an AppCat related documentation.

Reply:
I copied part of AppCat glossary cause I thought we want to align it to APPUiO Managed Kubernetes Services down the road, once we add more information. Sure, I will remove it.

Reply:
Yes, better get rid of this part again, this confuses too much and doesn't apply for this product.